### ISSUE TITLE
Improve styling for author page with regard to search bar

### ISSUE THREAD

**Issue by RayBB** [internetarchive/openlibrary#9742]
### Problem

There are several issues on the author search page in regards to the search bar.

#### Evidence / Screenshot
<img width="1107" alt="image" src="https://github.com/user-attachments/assets/035a09b8-6652-43cd-b0cd-0be2dc169bbd">
<img width="429" alt="image" src="https://github.com/user-attachments/assets/611249ab-cc64-4193-882c-7a74f0b6b6d8">
<img width="431" alt="image" src="https://github.com/user-attachments/assets/7d489500-4003-4d46-802c-2f85f7d3bf4e">

<img width="1093" alt="image" src="https://github.com/user-attachments/assets/93da5f1e-0717-4554-b420-20992d9ef646">
<img width="409" alt="image" src="https://github.com/user-attachments/assets/0e7fa4db-d9e4-4eba-ba5b-be4a4d8e1b97">

#### Relevant URL(s)
https://openlibrary.org/authors/OL31676A/Noam_Chomsky
https://openlibrary.org/authors/OL8278562A/Morgan_G._Ames

### Reproducing the bug

Issues on desktop:
* no spacing between the search bar and first result when there is no pagination
* When there is a lot of pagination it doesn't look too good either, not well aligned.

Issues on mobile:
* Spacing doesn't match the search results
* When there is pagination it is very chaotic

### Context

n/a

#### Proposal & constraints

We'll need someone with a careful eye at design to take a look at this one.

#### Stakeholders

<hr>

#### Instructions for Contributors

- Please [run these commands](https://github.com/internetarchive/openlibrary/wiki/Git-Cheat-Sheet#working-on-your-branch) to ensure your repository is up to date **before** [creating a new branch](https://github.com/internetarchive/openlibrary/wiki/Git-Cheat-Sheet#making-changes-and-creating-a-pull-request) to work on this issue and **each time after** pushing code to Github, because the pre-commit bot may add commits to your PRs upstream.

**Comment by sbwhitt** [comment]
I've been toying with this. I tried moving the navigation options above the page numbers and centered all of the pagination. The spacing problems are also easily fixable. Let me know if this looks any better and I can take this and put these changes up for review. 

![image](https://github.com/user-attachments/assets/f67d399c-baff-42ca-a8aa-47f1bc6bd9f2)

**Comment by RayBB** [comment]
@sbwhitt I think yours is a big improvement!

At the same time, I think  we should drastically simplify the design long term. Just have next/prev (and maybe first/last). 
For comparison sake I made a small album of how other sites do this: https://albums.ente.io/?t=Rc69oqWF#4Fb3FuJxeD1xqEy5kStywnrqTCMZXpwtUVXcnwJtnoPG

To make it beyond my opinion I'd recommend the following:
1. Collect analytics for how often each navigation button is clicked. To support drastic design changes.
2. If it's not too much work, make a PR for your layout changes (chichi don't remove buttons) because they're an improvement in any case.
3. Decide on removing buttons after a month or so about analytics 
4. Consider adding auto loading like most modern apps have (with a fallback to the old navigation for people without JS).

In short, I'd support you making a PR for your proposed changes :)

**Comment by sbwhitt** [comment]
I agree it could be simplified further. I would like to remove the text like some of the examples you showed, and maybe reduce the amount of page numbers as well. If you want to assign to me I can keep messing with it and put up a PR in a bit.

**Comment by RayBB** [comment]
@sbwhitt I assigned you. I know it'll be a slow next month or so as the team is away for the holidays but it would be awesome if you wanted to come up with some simpler mockups. 

Also if you want to do an analytics PR maybe we can sneak that out before the holidays to collect data and justify simplifying it. If you need an analytics example I have a recent PR for wikidata analytics.

**Comment by sbwhitt** [comment]
That's totally fine. I am just getting back into things so no rush. Also, I'd be up for that. I haven't worked with that before so an example would be helpful. Thanks for the help!

**Comment by RayBB** [comment]
https://github.com/internetarchive/openlibrary/pull/10069 is the example. Might have to put a little thinking into what they keys will be for each state. I think they probably should be relative like the words are the words but the others could be +1 +2 +3 or something like that to see how often people are skipping ahead or back more than one from where they currently are.

**Comment by cdrini** [comment]
Here are the stats!

![Image](https://github.com/user-attachments/assets/f5f16eae-8773-4a94-a45b-bec486492ab9)

**Comment by RayBB** [comment]
@sbwhitt fyi this if for the last 30 days.

**Comment by sbwhitt** [comment]
Awesome, thank you @cdrini !

**Comment by RayBB** [comment]
@sbwhitt are you still interested in working on this?

**Comment by sbwhitt** [comment]
@RayBB Unfortunately I haven't had the time lately. If anyone else is interested feel free to take over.

**Comment by RayBB** [comment]
@lokesh this has some info that can be useful if you wanna work on turning this into a lit component

**Comment by RayBB** [comment]
@lokesh should we use the new pagination component on author pages?
https://github.com/internetarchive/openlibrary/pull/11811

**Comment by rattnak** [comment]
Hi @RayBB. Could I be assigned to this? Would appreciate the opportunity to contribute!

**Comment by RayBB** [comment]
I assigned you to a different issue you asked about, but we wait until we hear back from Lokesh on how we should proceed here.

**Comment by lokesh** [comment]
There have been changes to this page since the issue was originally created with many of the originally noted issues having been addressed. 

> Issues on mobile:
> 
> Spacing doesn't match the search results
> When there is pagination it is very chaotic


<img width="458" height="225" alt="Image" src="https://github.com/user-attachments/assets/c4491a1f-39a3-4425-bcea-6fedabd3fdf3" />

_Note: The pagination update is queued up to being deployed but not on the live site._

The page could benefit from a layout pass, but the focus is currently on individual components (e.g. dropdown) or sets of components (e.g. search result item).

**Comment by RayBB** [comment]
Wow, I didn't know that you got this page fixed up with the new pagination component already. In that case, I think we can just close this issue, and when the time comes or if there are specific other issues, we can file them separately. Thanks for your input on this, and I look forward to seeing this deployed in production.
